src/app/clusters/tv-channel-server/tv-channel-server.cpp

```diff
@@ -39,24 +39,59 @@
  ******************************************************************************/
 
 #include <app/Command.h>
+#include <app/common/gen/af-structs.h>
+#include <app/common/gen/cluster-id.h>
+#include <app/common/gen/command-id.h>
 #include <app/util/af.h>
 
-bool emberAfTvChannelClusterChangeChannelCallback(chip::app::Command * commandObj, unsigned char *)
+EmberAfTvChannelInfo tvChannelClusterChangeChannel(std::string match);
+bool tvChannelClusterChangeChannelByNumber(uint16_t majorNumer, uint16_t minorNumber);
+bool tvChannelClusterSkipChannel(uint16_t count);
```

Comment:
zapt for callbacks as a followup, please.

Reply:
Added it to the existing [issue](https://github.com/project-chip/connectedhomeip/issues/8086).